### DISCUSSION TITLE
Does SImple login has support for blockchain based domains ?

### DISCUSSION THREAD

**Discussion by swapnil2905** [simple-login/app#1356] [Feature Requests]
Does simple login has support for web 3 based domains ? I have a web 3 based domain and would really love to link it to simple login
https://unstoppabledomains.com/

**Comment by nguyenkims** [comment]
@swapnil2905 currently SL doesn't support web3 domains as web3 doesn't seem to use SMTP.

**Reply by swapnil2905** [reply]
So, is there any plans for supporting the web 3 domains in the near future?
